pepper-apis/src/main/java/org/broadinstitute/ddp/route/CreateActivityInstanceRoute.java

```diff
@@ -68,8 +67,9 @@ public Object handle(Request request, Response response, ActivityInstanceCreatio
                 activityInstanceDao.bulkUpdateIsHiddenByActivityIds(participantId, true, Set.of(studyActivityId));
             }
 
+            Long parentInstanceId = null;
             String instanceGuid = activityInstanceDao
-                    .insertInstance(studyActivityId, operatorGuid, participantGuid, InstanceStatusType.CREATED, null)
+                    .insertInstance(studyActivityId, operatorGuid, participantGuid, parentInstanceId)
```

Comment:
`parentInstanceId` is always null. I am assuming handling children is part of DDP-5574/separate PR. Yes?

Reply:
Yup, that's right!